packages/kokoas-client/src/pages/projSearch/sections/result/details/paymentDetails/unissuedInvoiceAlert/saveReminder/getPlannedDepositDate.ts

```diff
@@ -0,0 +1,18 @@
+import format from 'date-fns/format';
+import { KAlertPurpose } from '../alertConfig';
+
+export const getPlannedDepositDate = ({
+  purpose,
+  paymentDate,
+}: {
+  purpose: KAlertPurpose
+  paymentDate: Date | null
+}) => {
+  if (purpose !== 'subsidy') {
+    return '';
+  } else {
+    if (!paymentDate) return '';
+
+    return format(paymentDate, 'yyyy-MM-dd');
+  }
```

Comment:
変更不要。

以下でも行けるかもしれません。

```suggestion
  if (purpose !== 'subsidy' || !paymentDate) {
    return '';
  }

  return format(paymentDate, 'yyyy-MM-dd');
```

Reply:
ご指摘ありがとうございます。
こちらの方がスマートだと思うので、修正させてもらいました。